agent/src/main/resources/pinpoint-real-env-lowoverhead-sample.config

```diff
@@ -167,6 +167,14 @@ profiler.jdbc.cubrid.rollback=false
 profiler.jdbc.dbcp=true
 profiler.jdbc.dbcp.connectionclose=false
 
+#
+# CASSANDRA
+#
+# Profile CASSANDRA.
+profiler.cassandra=true
```

Comment:
Plugin enable/disable is configured with `profiler.plugin.disable` property. You'd better remove this.